tests/rptest/tests/redpanda_test.py

```diff
@@ -50,11 +55,27 @@ def __init__(self,
             else:
                 num_brokers = 1
 
+        cloud_storage_access_key = test_context.globals.get(
+            self.GLOBAL_S3_ACCESS_KEY, None)
+        cloud_storage_secret_key = test_context.globals.get(
+            self.GLOBAL_S3_SECRET_KEY, None)
+
+        # Enable S3 if AWS creds were given at globals
+        if cloud_storage_access_key is not None and cloud_storage_secret_key is not None:
+            if self.si_settings is not None:
+                self.si_settings.cloud_storage_access_key = cloud_storage_access_key
+                self.si_settings.cloud_storage_secret_key = cloud_storage_secret_key
+                self.si_settings.endpoint_url = None  # None so boto auto-gens the endpoint url
+                self.si_settings.cloud_storage_disable_tls = False  # SI will fail to create archivers if tls is disabled
+        else:
+            self.logger.debug('Invalid AWS creds')
```

Comment:
I don't think this is really invalid -- they're allowed to just leave them out when using minio, right?

Reply:
correct and changed.